.env.template

```
@@ -1,2 +1,3 @@
 RESEND_API_KEY=<YOUR API KEY>
-RECIPIENT_EMAIL=<YOUR EMAIL>
\ No newline at end of file
+RECIPIENT_EMAIL=<YOUR EMAIL>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RECIPIENT_EMAIL key should go before the RESEND_API_KEY key

(UnorderedKey)